src/pages/index.tsx

```diff
@@ -92,8 +83,27 @@ const Home: NextPage<DefaultSeoInfo> = (props) => {
 
 export async function getServerSideProps(
 	context: GetServerSidePropsContext
-): Promise<GetServerSidePropsResult<DefaultSeoInfo>> {
-	return getDefaultServerSideProps(context);
+): Promise<GetServerSidePropsResult<HomepageProps>> {
+	let title: string | null = null;
+	try {
+		const [contentPage] = await Promise.allSettled([
+			ContentPageService.getBySlug(('/' + context.query.slug) as string),
+		]);
+
+		if (contentPage.status === 'fulfilled') {
+			title = contentPage.value?.title || null;
+		}
```

Comment:
No need to use the Promise.allSettled if there is only one promise to resolve
```suggestion
		const contentPage = await ContentPageService.getBySlug(('/' + context.query.slug) as string));
		title = contentPage.value?.title || null;
```